### PR TITLE
KEYCLOAK-4412: Allow multiple theme directories

### PR DESCRIPTION
**Pull request by ssilvert**

### CODE REVIEW

File: testsuite/integration-arquillian/tests/other/console/pom.xml [hunk 1]
```diff
@@ -32,7 +32,8 @@
     <name>Admin Console UI Tests</name>
 
     <properties>
-        <keycloak.theme.dir>${auth.server.home}/themes</keycloak.theme.dir>
+        <keycloak.theme.dir>${auth.server.home}/themes-primary</keycloak.theme.dir>
+        <developer.theme.dir>${project.basedir}/../../../../../themes/src/main/resources/theme</developer.theme.dir>
```
Comment (stianst) [inline testsuite/integration-arquillian/tests/other/console/pom.xml]: Is this not changing the server that is tested? There seems to be a chance here that resources not copied by mistake when building the server would be loaded directly from themes/src and the bug wouldn't be caught during testing.

**Review by stianst — CHANGES_REQUESTED** [review]

File: testsuite/integration-arquillian/tests/other/console/pom.xml (hunk 1 above)
Comment (ssilvert) [inline testsuite/integration-arquillian/tests/other/console/pom.xml]: I'm not sure I understand your question.  It's not changing the server being tested.  

Line 35 changes the directory where the custom themes are copied and later found at runtime.
Line 36 lets the "base" and "keycloak" themes be read from the development directory.

Comment (stianst) [inline testsuite/integration-arquillian/tests/other/console/pom.xml]: Exactly what I'm saying. You're reading the themes from a different directory, so the setup doesn't match an actual real installation.

Comment (ssilvert) [inline testsuite/integration-arquillian/tests/other/console/pom.xml]: Ah.  I thought you meant I was somehow changing which server instance was being tested.  But in this case I don't understand your objection.   We have plenty of other tests that use the default configuration.

If you like, I can just move those two lines to a different profile.

**Pull request closed without merging**